src/schema/rules/filenames.yaml

```diff
@@ -0,0 +1,27 @@
+---
+root:
+  name: "root"
```

Comment:
name in this context doesn't make sense.

Reply:
```suggestion
```
Yeah, it muddies the waters so to speak; testing suggesting nothing for you to commit.